apps/web/src/components/execution-detail/helpers.ts

```diff
@@ -78,6 +79,10 @@ export const getLogoByType = (
     return Mobile;
   }
 
+  if (type === StepTypeEnum.CUSTOM) {
+    return CurlyBrackets;
+  }
+
```

Comment:
Adding the missing icon for custom

<img width="1539" alt="image" src="https://github.com/novuhq/novu/assets/32132657/61551dcb-22c5-42da-bc95-edb17a130612">